Samples~/AvatarCreatorSamples/AvatarCreatorWizard/Scripts/Utils/AssetTypeHelper.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using ReadyPlayerMe.AvatarCreator;
+public static class AssetTypeHelper
```

Comment:
Minor but I would suggest wrapping this class in the ReadyPlayerMe.Samples.AvatarCreatorWizard name space like we do in the other sample scripts. It's useful to prevent potential class naming collisions, although not super likely.  

EG 
```cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Reflection;

namespace ReadyPlayerMe.Samples.AvatarCreatorWizard {

// class code here

}
```